tests/cli/test_worker.py

```diff
@@ -215,7 +215,7 @@ async def test_worker_discovers_work_pool_type(
     assert workers[0].name == "test-worker"
 
 
-async def test_worker_errors_if_no_type_and_non_existent_work_pool():
+async def test_worker_if_no_type_and_non_existent_work_pool():
```

Comment:
Can you add a check to this test to verify that the expected work pool was created by the worker?